enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionDefaultView/InteractiveQuestionDefaultView.tsx

```diff
@@ -193,7 +193,17 @@ export const InteractiveQuestionDefaultView = ({
       <Box className={InteractiveQuestionS.Main} p="sm" w="100%" h="100%">
         <Box className={InteractiveQuestionS.Content}>
           {isEditorOpen ? (
-            <InteractiveQuestion.Editor onApply={closeEditor} />
+            <>
+              {/* Use the same horizontal padding as https://github.com/metabase/metabase/blob/98e2dcc7c8c7c9147f4a787cc7ed36eddde9c080/frontend/src/metabase/querying/notebook/components/Notebook/Notebook.tsx#L48 */}
+              {/* Vertical padding matches the visualization header above*/}
+              {/* If we don't conditionally render this button, it will be shown twice after visualize the query once. */}
+              {!queryResults && (
+                <Box px={{ base: "1rem", sm: "2rem" }} pt="md">
+                  <InteractiveQuestion.BackButton />
+                </Box>
+              )}
```

Comment:
This is only for dashboards I assume, as we show "Back to..." for questions.

![Image](https://github.com/user-attachments/assets/38f57e7d-c8c0-4f14-be89-30453cc494ac)

Reply:
Oh sheesh, I need to fix that. Thanks for flagging that.